test/unit/awssigv4signer.test.js

```diff
@@ -0,0 +1,407 @@
+/*
```

Comment:
This should match the file structure and naming in `lib`, so `test/unit/lib/aws/AwsSigv4Signer.test.js`

Reply:
makes sense